.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,10 @@
-Fixes #
-
 <!--
 Please follow the naming conventions in the "guidelines for contributing" link above or the pull request may be rejected.
 
 Also, do consider outlining the solution taken as doing so will likely help in the reviewing process
 (e.g., when the pull request involves non-trivial changes)
 -->
+
+Fixes #
+
+**Outline of Solution**
```

Comment:
Although quite obvious, you may want to add an explanation to this section too, something like "Tell us how you solve the issue" or something along that line.